STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -359,7 +361,7 @@ The Order object that is created on the IdO:
 
```

Comment:
Wait, both here and in the non-STAR section we have two occurrences each of "the Order object created on/by the IdO", with the contradictions that can be expected.

Specifically, the `delegation` attribute must be included on the NDC-IdO leg (both Order request and Order returned) but not on the IdO-CA leg.

Reply:
IIUC, the (apparent) contradiction / confusion would be sorted by your suggestion below, correct?

Reply:
Yes.